components/net/protocols/mod.rs

```diff
@@ -47,6 +48,13 @@ pub trait ProtocolHandler: Send + Sync {
     fn is_fetchable(&self) -> bool {
         false
     }
+
+    /// Specify if this custom protocol is considered secure context
+    ///
+    /// Note: this only works for bypassing mixed content checks right now
```

Comment:
Is there a specification for this? If so it might be good to link to it.

```suggestion
    /// Specify if this custom protocol is usable in secure contexts.
    ///
    /// Note: this only works for bypassing mixed content checks right now
```

Reply:
about the secure context, https://www.w3.org/TR/secure-contexts/ or https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts, which one do you prefer?